--- parsehtab.c	2012-07-27 16:00:45.000000000 +0100
+++ parsehtab.c.corrected	2014-04-21 19:29:50.950296145 +0100
@@ -15,7 +15,7 @@
    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.  */
 
-#include "config.h"
+#include "../config.h"
 #include <stdio.h>
 #include <ctype.h>
 #include <sys/types.h>
@@ -23,7 +23,7 @@
 #include <time.h>
 #include "defaults.h"
 #include "networkincl.h"
-#include "incl_unix.h"
+#include "../src/hdrs/incl_unix.h"
 #include "remote.h"
 #include "hostedit.h"
 
