--- xhostedit.c	2013-08-21 08:31:52.000000000 +0100
+++ xhostedit.c.corrected	2014-04-21 19:31:21.556932852 +0100
@@ -22,13 +22,13 @@
 #include <time.h>
 #include <pwd.h>
 #include <ctype.h>
-#include "config.h"
-#include "defaults.h"
-#include "incl_unix.h"
+#include "../config.h"
+#include "../src/hdrs/defaults.h"
+#include "../src/hdrs/incl_unix.h"
 #include "networkincl.h"
 #include "remote.h"
 #include "hostedit.h"
-#include "src/hdrs/files.h"
+#include "../src/hdrs/files.h"
 
 static	char	rcsid2[] = "@(#) $Revision: 1.10 $";
 
